airflow/providers/microsoft/azure/hooks/azure_fileshare.py

```diff
@@ -96,6 +96,26 @@ def list_directories_and_files(self, share_name, directory_name=None, **kwargs):
         """
         return self.get_conn().list_directories_and_files(share_name, directory_name, **kwargs)
 
+    def list_files(self, share_name, directory_name=None, **kwargs):
```

Comment:
We usually add tests for hook methods. Here you should add tests for the hook methods.
https://github.com/apache/airflow/tree/master/tests/providers/microsoft/azure/hooks

Reply:
sure thing, done